utils/constants.py

```diff
@@ -51,6 +51,7 @@ class DataType(ValueEnum):
     POSTAL_CODE = "postalCode"
     POSTAL_CODE_VARIANT = "postal_code"
```

Comment:
I don't think this variant is ever part of an address object anymore so I was able to ignore it I think.

Reply:
I think it still exists in the internal api

Reply:
Yep it's still in some places in the internal API but not part of any Address definition more one off fields.  It should use the new library too for those individual fields https://github.com/transcom/milmove_load_testing/pull/96/files#diff-13ff359e13fc9a6114896673b1c3cbb9ad704f9e7a5a3818f0cbe5786cfc2848R100